lib/globalize/active_record/query_methods.rb

```diff
@@ -63,6 +71,25 @@ def join_translations(relation = self)
           relation.with_translations_in_fallbacks
         end
       end
+
+      private
+
+      def parse_translated_order(opts)
+        case opts
+        when Hash
+          ordering = opts.map do |column, direction|
+            klass = translated_column?(column) ? translation_class : self
+            klass.arel_table[column].send(direction)
+          end
+          order(ordering).order_clauses
+        when Symbol
+          translated_column_name(opts) if translated_attribute_names.include?(opts)
+        end
```

Comment:
I believe it would be prudent to ensure that the case statement has a default case